Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
+FROM nvidia/cuda:11.1-cudnn8-devel-ubuntu18.04
```

Comment:
@Saurav-D Please revert this. This change will disable GPU support for pytorch as the package in pypi only supports CUDA 10 for pytorch 1.4.x.

Reply:
@adason Is there any issue about this on pytorch? I launched train pipeline using this and it ran successfully using GPUs.

Reply:
Also, pip install downgrades cuda version to 10.1 as can be seen in the following:
```
# python
Python 3.7.5 (default, Nov  7 2019, 10:50:52) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> print(torch.version.cuda)
10.1
```

I ran the above inside the docker image of the latest build.